platform/linux_platform_test.go

```diff
@@ -4136,15 +4136,17 @@ unit: sectors
 	})
 
 	Describe("GetDefaultNetwork", func() {
-		It("delegates to the defaultNetworkResolver", func() {
-			defaultNetwork := boshsettings.Network{IP: "1.2.3.4"}
-			fakeDefaultNetworkResolver.GetDefaultNetworkNetwork = defaultNetwork
-
-			network, err := platform.GetDefaultNetwork()
-			Expect(err).ToNot(HaveOccurred())
-
-			Expect(network).To(Equal(defaultNetwork))
-		})
+		for _, value := range []bool{true, false} {
```

Comment:
Can `value` be renamed to something that expresses intent (like whatever is chosen as the name of the `bool` where `GetDefaultNetwork()` is defined?